packages/devtools_app/lib/src/debugger/flutter/codeview.dart

```diff
@@ -69,9 +69,6 @@ class _CodeViewState extends State<CodeView> {
 
     if (widget.script != oldWidget.script) {
       _updateLines();
-    }
-
-    if (widget.stack != oldWidget.stack) {
```

Comment:
can you comment why we want to always update the paused positions even if the stack has not changed?